doc/api/buffer.md

```diff
@@ -1994,8 +1994,11 @@ added: v0.1.90
 * `string` {string} String to write to `buf`.
 * `offset` {integer} Number of bytes to skip before starting to write `string`.
   **Default:** `0`.
-* `length` {integer} Maximum number of bytes to write. **Default:**
-  `buf.length - offset`.
+* `length` {integer} Maximum number of bytes to write (written bytes will not
+  exceed `buf.length - offset`). **Default:** `buf.length - offset`.
+  If the buffer has sufficient space from the offset, the string is written
+  up to `length`. If the buffer is short in space, only `buf.length - offset`
+  bytes are written.
```

Comment:
@HarshithaKP The last three sentences all appear to be saying the same thing. Can I suggest that everything after the `**Default:**` can be removed, and instead by explained through examples below?

(Also, `**Default:**` should be the last item in the description here anyway.)

Reply:
@addaleax, thanks for the suggestion. Added example to it. PTAL.